src/OpenTelemetry.Exporter.Prometheus/Implementation/PrometheusMetricsFormatHelper.cs

```diff
@@ -27,8 +27,6 @@ namespace OpenTelemetry.Exporter.Prometheus
 {
     internal static class PrometheusMetricsFormatHelper
     {
-        public const string ContentType = "text/plain; version = 0.0.4";
```

Comment:
why not update this?

Reply:
I think this is not going to be used in other places, making it inline increases readability.